### PR TITLE
[bulk] Validation: Metadata and Absolute Links

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

File: docs-ref-services/preview/cosmos-readme.md [hunk 1]
```diff
@@ -300,26 +299,26 @@ or contact [opencode@microsoft.com][coc_contact] with any additional questions o
...
-[cosmos_docs]: https://docs.microsoft.com/azure/cosmos-db/introduction
-[jdk]: https://docs.microsoft.com/java/azure/jdk/
+[cosmos_docs]: /azure/cosmos-db/introduction
+[jdk]: /java/azure/jdk/
 [slf4j]: http://www.slf4j.org/
 [maven]: https://maven.apache.org/
 [cosmos_maven]: https://search.maven.org/artifact/com.azure/azure-cosmos
 [cosmos_maven_svg]: https://img.shields.io/maven-central/v/com.azure/azure-cosmos.svg
 [cla]: https://cla.microsoft.com
-[coc]: https://opensource.microsoft.com/codeofconduct/
```
Comment (MelanieHom) [inline docs-ref-services/preview/cosmos-readme.md:312]: Is it okay to remove a different absolute link?

Comment (v-emilypr) [reply docs-ref-services/preview/cosmos-readme.md:312]: Good catch, thank you!

**Comment by v-emilypr** [comment]
**Global effort to fix build validation errors**

@kushagraThapar, @Shanfang 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by v-emilypr** [comment]
@kushagraThapar, should we back out all changes to absolute links on readme.mds on the azure-docs-sdk-java repo? The validation team is working to clear the build warnings from this repo. I will loop in our content manager @v-andreaco so that we can coordinate how this should be best handled.

**Comment by v-andreaco** [comment]
@kushagraThapar 

Thanks so much for your patience! We have a large team working on these issues, so sometimes announcements slip through. I really appreciate you taking the time to explain, and we'll send out a stronger note to the team to back these changes up.

I think I asked in another PR if metadata changes will also be overwritten, and I don't remember seeing if that was answered. Should our team revert **all** changes to these files, or just the content edits, and metadata changes can go through?

Thanks again for being so helpful!! I know your time is valuable, and we really appreciate you taking the time to help us!

**Comment by kushagraThapar** [comment]
@v-andreaco - metadata changes can still go through, but my concern is this:

We release our SDKs once every month (on usual cadence), and sometimes twice in a month. With every release, our docs go out as part of the release process, which I am not sure will keep the meta data changes that your team is making or not. 

One way to find out is to merge one of the PRs for the metadata changes, and see in May 2nd week, (once we release), if they will be kept or erased. Let me know if this works or not !

**Comment by v-andreaco** [comment]
@kushagraThapar 

That sounds like a great idea! Can we test with this one?

**Comment by v-emilypr** [comment]
@kushagraThapar and @v-andreaco I have backed out the content changes and left metadata in place for testing on this PR.

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks!

**Comment by kushagraThapar** [comment]
@v-andreaco - yes, I can merge this, thanks!

**Pull request merged**